Cargo.toml

```
@@ -4,27 +4,16 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-riscv-rt   = "0.9.0"
-shared-bus = { version = "0.2.4",  features = ["xtensa"] }
-shtcx      = { version = "0.10.0", optional = true }
-smart-leds = { version = "0.3.0",  optional = true }
-
-[dependencies.esp32c3-hal]
-git     = "https://github.com/esp-rs/esp-hal/"
-package = "esp32c3-hal"
-rev     = "1655e36"
-
-[dependencies.esp-backtrace]
-git      = "https://github.com/esp-rs/esp-backtrace"
-rev      = "9eb1cb4"
-features = ["esp32c3", "exception-handler", "panic-handler", "print-jtag-serial"]
-
-[dependencies.icm42670]
-git      = "https://github.com/jessebraham/icm42670/"
-rev      = "eb0483b"
-optional = true
+esp-backtrace = { version = "0.2.0", features = ["esp32c3", "exception-handler", "panic-handler", "print-jtag-serial"] }
+esp-println   = { version = "0.3.0", features = ["esp32c3"] }
+esp32c3-hal   = "0.1.0"
```

```suggestion
esp32c3-hal   = "0.2.0"
```

I've just today published a new version of this package (there were some issues with the previous version), so would you mind updating this please?